modules/apps/frontend-taglib/frontend-taglib/src/main/java/com/liferay/frontend/taglib/servlet/taglib/ScreenNavigationTag.java

```diff
@@ -80,6 +80,10 @@ public Object getContext() {
 		return _context;
 	}
 
+	public boolean getDisableAriaCurrent() {
+		return _disableAriaCurrent;
+	}
```

Comment:
To avoid the awkward getter, attribute names should not be verbs. We can name the attribute `ariaCurrentEnabled`. As it is a `boolean`, getter should also start with `is`. Two changes would make this `isAriaCurrentEnabled`.

Another option should be to set HTML attribute through the tag attribute, `ariaCurrent` string.

Reply:
Hi @markocikos! 😄 yeah, I think the second option you suggested is the best one. At the same time I sent this PR I opened [this ticket in Clay](https://github.com/liferay/clay/issues/5307) to make aria-current also optional in Clay navigation components. Looking at the solution that Matu implemented, I agree that we should do it in a similar way in the taglib. I'll do the update.